apps/web/tailwind.config.js

```diff
@@ -34,8 +37,8 @@ module.exports = {
         20: '20px',
       },
       boxShadow: {
-        medium: '0px 1px 10px rgba(0, 0, 0, 0.15)',
-        large: '0px 5px 40px rgba(0, 0, 0, 0.15)',
+        medium: 'var(--shadowMedium)',
```

Comment:
💯

Reply:
still need to reconcile these with `shadow-md` and `shadow-lg` which are _also_ used throughout the app and are quite different from these guys.